lib/interpolateName.js

```diff
@@ -3,11 +3,26 @@
 const path = require('path');
 const emojisList = require('emojis-list');
 const getHashDigest = require('./getHashDigest');
+const {
+  DigitUglify,
+  AlphabetUglify,
+  WordUglify,
+  FileUglify,
+  CssUglify,
+} = require('uglify-set');
 
 const emojiRegex = /[\uD800-\uDFFF]./;
 const emojiList = emojisList.filter((emoji) => emojiRegex.test(emoji));
 const emojiCache = {};
 
+const uglify = {
+  digit: DigitUglify(),
+  alphabet: AlphabetUglify(),
+  word: WordUglify(),
+  file: FileUglify(),
+  css: CssUglify(),
+};
```

Comment:
continuously https://github.com/webpack/loader-utils/pull/175#discussion_r476627900 by @evilebottnawi

Reply:
Still memory leak

Reply:
Yes this changes aren't for fixing memory leak.
I told like this in the previous comment:
> Maybe I'm not catch the point now.
> Can you explain it more detail?
Can you tell cause of memory leak or test method?

Reply:
@TroyTae just run your projects on very very very big code base, you can even write test in own repository

Reply:
![image](https://user-images.githubusercontent.com/20078201/91466571-4a8cca00-e8ca-11ea-8572-97aac82ea976.png)
(Capture snapshot after executing 9 million times)
Hmmm.. Can you tell me how do you test it?
It seems garbage collection works well...

Reply:
You need to run your tool in watch mode

Reply:
Above capture is tested in watch mode.
I made a webapp and run it using `webpack-dev-server`.

Reply:
I added force reset code when over 2 ** 16.
And this is the test code:
https://github.com/TroyTae/uglify-set/blob/master/__tests__/test-util.js#L13-L25
Can you check this again?

Reply:
@evilebottnawi Anything is blocking this?